model_utils/tracker.py

```diff
@@ -66,14 +66,14 @@ def current(self, fields=None):
 
     def has_changed(self, field):
         """Returns ``True`` if field has changed from currently saved value"""
-        if field in self.fields:
```

Comment:
@furious-luke shall we keep the if in both methods? Can you add a test for your change?